src/components/Table/Table/__stories__/table.stories.mdx

```diff
@@ -145,43 +157,89 @@ Using skeleton
 
 <Canvas>
   <Story name="Loading">
-    <Table errorState={<TableErrorState />} emptyState={<TableEmptyState />} columns={emailColumns}>
-      <TableHeader>
-        {emailColumns.map((headerCell, index) => (
-          <TableHeaderCell key={index} title={headerCell.title} />
-        ))}
-      </TableHeader>
-      <TableBody>
-        {[...new Array(5)].map((key, index) => (
-          <TableRow key={index}>
-            <TableCell>
-              <Skeleton type={Skeleton.types.TEXT} height={20} width={index % 2 === 0 ? 140 : 70} />
-            </TableCell>
-            <TableCell>
-              <Skeleton type={Skeleton.types.TEXT} height={20} width={index % 2 === 0 ? 200 : 120} />
-            </TableCell>
-            <TableCell>
-              <Skeleton type={Skeleton.types.CIRCLE} height={24} width={24} />
-            </TableCell>
-            <TableCell>
-              <Skeleton type={Skeleton.types.TEXT} height={20} />
-            </TableCell>
-            <TableCell>
-              <Skeleton type={Skeleton.types.TEXT} height={20} />
-            </TableCell>
-          </TableRow>
-        ))}
-      </TableBody>
-    </Table>
+    <TableContainer>
+      <Table
+        dataState={{ isLoading: true }}
+        errorState={<TableErrorState />}
+        emptyState={<TableEmptyState />}
+        columns={emailColumns}
+      >
+        <TableHeader>
+          {emailColumns.map((headerCell, index) => (
+            <TableHeaderCell key={index} title={headerCell.title} />
+          ))}
+        </TableHeader>
+        <TableBody>
+          {emailTableData.map(rowItem => (
+            <TableRow key={rowItem.id}>
+              <TableCell>{rowItem.sentOn}</TableCell>
+              <TableCell>{rowItem.subject}</TableCell>
+              <TableCell>
+                <TableAvatar text={rowItem.sentBy} />
+              </TableCell>
+              <TableCell>
+                <Label text={rowItem.status} color="positive" isAnimationDisabled />
+              </TableCell>
+              <TableCell>{rowItem.emailsSent}</TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
   </Story>
 </Canvas>
 
 ## Scroll
 
-This is an example of a table with 5000 rows
+Table with both vertical and horizontal scroll
 
 <Canvas>
   <Story name="Scroll">
+    {
+      {
+        /* styles.scrollTable = height: 220px; width: 800px to enforce scrolling container */
+      }
+    }
+    <TableContainer className={styles.scrollTable}>
```

Comment:
I see that this is the only place that we're actually placing something on the `TableContainer` container.
All the rest are empty components such as `<TableContainer> ... </TableContainer>`

Are we sure we need to wrap the Table with another component?

Reply:
If we're not wrapping it, and we're styling Table (and assigning it overflow) straight, the Table cannot fit under no container.
I checked with more libraries and didn't find one that do not wrap it inside their design system.

The reason is that the overflow with border-radius and others should be on the wrapper, and if we're not wrapping it ourselves we are asking people to implement it themselves